frontend/src/app/api/auth/callback/route.ts

```
@@ -11,7 +11,9 @@ export async function GET(request: NextRequest) {
   try {
     await createSession(token);
   } catch (_e) {
+    console.error("Error creating session for token", { token });
```

This is happening server side so we do actually want to log any details that might help us troubleshoot, not just eat the error.